core-ui/src/components/Extensibility/components/CodeViewer.js

```diff
@@ -16,33 +16,36 @@ export function CodeViewer({ value, structure, schema }) {
     let language = structure.language || detectLanguage(value);
     let parsedValue = '';
 
-    try {
-      switch (language) {
-        case 'yaml':
-          parsedValue = jsyaml.dump(value);
-          break;
-        default:
-          //this includes JSON and other languages
-          parsedValue = stringifyIfObject(value);
+    if (value) {
```

Comment:
`0` / `false` are proper values to display, this conditions should be only for empty values